src/main/java/uk/gov/pay/adminusers/model/Service.java

```diff
@@ -30,6 +30,7 @@
     private boolean redirectToServiceImmediatelyOnTerminalState;
     private boolean collectBillingAddress;
     private GoLiveStage goLiveStage;
+    private boolean experimentalFeaturesEnabled;
```

Comment:
Will it for all experimental features ? as the name implies. It is possible that different services try different experimental features at a time and we might accidentally enable for the services not needed.

Reply:
in the ticket we talk about potential extension and migration to a JSONB field I think?

Reply:
Yeah as described in https://github.com/alphagov/pay-adminusers/pull/698 this is just a generic "opt in" to whatever the developers are up to -- if we need more permanent control over a feature (like redirect on failure) it should be its own flag. Any features that enable themselves with this flag should be temporary and eventually open to all users. 

If we need more fine grained control over temporary features then we could use JSONB or something as suggested.

Reply:
As discussed (offline), we will be fine with this for current feature. Devs should be aware of the impact of enabling this feature and when adding a new feature, if another feature is in experimental stage or being released to services in phases. We might need to  track the services with the flag set so product can look back.